### ISSUE TITLE
torch.utils.serialization.load_lua was removed in pytorch v1.0

### ISSUE THREAD

**Issue by cclauss** [NVlabs/DG-Net#13]
pytorch/pytorch#15307

Also, __Vgg16()__ is defined at https://github.com/NVlabs/DG-Net/blob/master/networks.py#L758 but is not imported into this file.

[flake8](http://flake8.pycqa.org) testing of https://github.com/NVlabs/DG-Net on Python 3.7.1

$ __flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics__
```
./utils.py:229:18: F821 undefined name 'load_lua'
        vgglua = load_lua(os.path.join(model_dir, 'vgg16.t7'))
                 ^
./utils.py:230:15: F821 undefined name 'Vgg16'
        vgg = Vgg16()
              ^
./utils.py:234:11: F821 undefined name 'Vgg16'
    vgg = Vgg16()
          ^
./networks.py:577:25: F821 undefined name 'SpectralNorm'
            self.conv = SpectralNorm(nn.Conv2d(input_dim, output_dim, kernel_size, stride, dilation=dilation, bias=self.use_bias))
                        ^
4     F821 undefined name 'SpectralNorm'
4
```
__E901,E999,F821,F822,F823__ are the "_showstopper_" [flake8](http://flake8.pycqa.org) issues that can halt the runtime with a SyntaxError, NameError, etc. These 5 are different from most other flake8 issues which are merely "style violations" -- useful for readability but they do not effect runtime safety.
* F821: undefined name `name`
* F822: undefined name `name` in `__all__`
* F823: local variable name referenced before assignment
* E901: SyntaxError or IndentationError
* E999: SyntaxError -- failed to compile a file into an Abstract Syntax Tree

**Comment by layumi** [comment]
Hi @cclauss 

Thanks for your advice. The VGG is left due to the inception loss. I will consider to remove it in the next version. Thanks a lot.
